test/CMakeLists.txt

```diff
@@ -30,6 +31,7 @@ add_tests(
     function_traits
     intrusive_forward_list
     intrusive_list
+    intrusive_list_properties
```

Comment:
**suggestion**: I'd rather not separate example-based and property-based tests; can we put them all in one target?

Reply:
i can do that